documentation/README.md

```diff
@@ -871,13 +872,10 @@ and returns `undefined` if the `predicate` is false.
 ```js
 const ifThen = require('1-liners/ifThen');
 
-const eq = (a, b) => a === b;
-const add = (a, b) => a + b;
-const sub = (a, b) => a - b;
```

Comment:
[this](https://github.com/1-liners/1-liners/pull/164/files/d14dccb2f1ee91f4621c5eaea84b2041396c8afc#r74090305) too?